src/utils/env.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @flow
+ * @file File for some simple environment-related utilities
+ * @author Box
+ */
+
+/**
+ * Return true if we are currently running in a test or development
+ * environment.
+ *
+ * @return {boolean} true if we are running in a test or dev environment
+ */
+export default function isDevEnvironment() {
+    return (process && process.env.NODE_ENV === 'test') || process.env.NODE_ENV === 'dev';
```

Comment:
It looks like the second clause could still throw undefined, since it's checking `process.env` without checking `process`.